scripts/entrypoint.sh

```
@@ -6,14 +6,14 @@
 #
 ############################################################################
 
-INIT_PHI=${INIT_PHI:=True}
+INIT_PHI=$""{INIT_PHI:=True}
```

Shellcheck

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)